inc/modules/export/class-export.php

```diff
@@ -825,8 +825,8 @@ static function postExport() {
 
 		if ( is_countable( $validation_warning ) && count( $validation_warning ) ) {
 			// Validation warning
-			$exportoptions = get_option( 'pressbooks_export_options' );
-			if ( 1 === (int) $exportoptions['email_validation_logs'] || is_super_admin() ) {
+			$exportoptions = get_option( 'pressbooks_export_options', [] );
+			if ( ! empty( $exportoptions ) && 1 === (int) $exportoptions['email_validation_logs'] || is_super_admin() ) {
```

Comment:
It throws a warning in the logs when `email_validation_logs` key is not present in `$exportoptions`